lang/kotlin/kotlin.py

```diff
@@ -0,0 +1,186 @@
+from talon import Context, Module, actions, settings
+
+ctx = Context()
+mod = Module()
+ctx.matches = r"""
+code.language: kotlin
+"""
+
+# Java Modifies
+kotlin_modifiers = {
+    "var": "var",
+    "val": "val",
+    "lateinit": "lateinit",
+    "public": "public",
+    "private": "private",
+    "protected": "protected",
+    "static": "static",
+    "synchronized": "synchronized",
+    "volatile": "volatile",
+    "transient": "transient",
+    "abstract": "abstract",
+    "interface": "interface",
+    "final": "final",
+}
+
+mod.list("kotlin_modifier", desc="Kotlin Modifiers")
+ctx.lists["self.kotlin_modifier"] = kotlin_modifiers
```

Comment:
I think we should probably modernize this to use the code_keywords stuff? See JavaScript for an example 

https://github.com/talonhub/community/blob/main/lang/tags/keywords.talon